sql/synth_recipes.sql

```diff
@@ -2539,7 +2539,7 @@ INSERT INTO `synth_recipes` VALUES (34512,0,0,0,0,0,92,0,0,0,0,4099,4241,1369,91
 -- INSERT INTO `synth_recipes` VALUES (34514,0,2011,0,0,0,93,0,0,0,0,4098,4240,843,843,843,843,843,843,2145,2145,2145,2145,2145,2145,1417,1417,1417,1417,18,24,30,36,'Giant Bird Fletchings');
 INSERT INTO `synth_recipes` VALUES (34515,0,0,0,48,0,93,0,0,0,0,4099,4241,682,682,823,828,1699,1764,0,0,15392,15394,15394,15394,1,1,1,1,'Hachiman Hakama');
 INSERT INTO `synth_recipes` VALUES (34516,0,0,0,0,0,93,0,0,0,0,4098,4240,843,843,0,0,0,0,0,0,1417,1417,1417,1417,6,8,10,12,'Giant Bird Fletchings');
-INSERT INTO `synth_recipes` VALUES (34517,0,0,0,0,50,93,50,0,0,0,4099,4241,828,828,855,2152,2340,2470,2476,0,2449,2450,2450,2450,1,1,1,1,'Cursed Hat');
+INSERT INTO `synth_recipes` VALUES (34517,0,0,0,0,50,93,42,0,0,0,4099,4241,828,828,855,2152,2340,2470,2476,0,2449,2450,2450,2450,1,1,1,1,'Cursed Hat'); -- wiki, BG, jpwiki, AH checked 24/Nov/2021
```

Comment:
BG Wiki has 50. Pedia has 42. AH has ??? - In case of conflicting information, we go for highest level.

If you have the JPwiki values, send the link, pls. Couse i gave up on navigating that place.

Reply:
https://wiki-ffo-jp.translate.goog/html/12224.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=nui&_x_tr_sch=http

JP has 42 like Pedia and its unusual for FFXI to have both subcrafts at the same level.

Reply:
Source citations like this make Teo happy.